openedx/core/lib/command_utils.py

```diff
@@ -8,17 +8,18 @@
 from opaque_keys.edx.keys import CourseKey
 
 
-def get_mutually_exclusive_required_option(options, option_1, option_2):
+def get_mutually_exclusive_required_option(options, *selections):
     """
     Validates that exactly one of the 2 given options is specified.
     Returns the name of the found option.
     """
-    validate_mutually_exclusive_option(options, option_1, option_2)
 
-    if not options.get(option_1) and not options.get(option_2):
-        raise CommandError('Either --{} or --{} must be specified.'.format(option_1, option_2))
+    selected = [sel for sel in selections if options.get(sel)]
+    if len(selected) != 1:
+        selection_string = u', '.join('--{}'.format(selection) for selection in selections)
 
-    return option_1 if options.get(option_1) else option_2
+        raise CommandError(u'Must specify exactly one of {}'.format(selection_string))
+    return selected[0]
```

Comment:
This was updated to make room for configuring command via django admin (in the next PR), then the user would specify one of `--courses`, `--all_courses`, `--use_admin_config`.

In the process, I discovered that this is untested code, so I added tests for it, and had to change a few error messages in tests, since the error message assumed two values.